GuessSTARTTLSPolicies.py

```diff
@@ -0,0 +1,681 @@
+#!/usr/bin/python
```

Comment:
Something like this might be handy.  It would need clean-up, simplification and to use the ssl library consistent with the new Certbot stuff.  Even though we can't reasonably distribute such a config file (as it is horribly flawed in so many ways), we could still use to it do spot checks.

We could make it do just this:
Given a list of X domains, go get fingerprints, and tls settings, and write it out in the standard format, for testing only.